### DISCUSSION TITLE
Development with Sveltekit

### DISCUSSION THREAD

**Discussion by ArslanT57** [specialdoom/proi-ui#41] [Q&A] [answered]
Hi,
I am using proi-ui with my sveltekit app. I am having this error notification in VS Code. Here is a screenshot 
![image](https://user-images.githubusercontent.com/27070690/171410749-0816a846-97ef-4431-a2a4-ddaed10aab2d.png)
But the website is working fine. The button is properly rendered, as seen in the image below.
![image](https://user-images.githubusercontent.com/27070690/171411029-850b8b85-aef5-4f8a-b8b2-2b295ea822ce.png)

What is the issue, is it just a false alarm which it seems to be or is there something I am missing here. 

Steps to reproduce this error.

1. Use VS Code
2. Create a new sveltekit app
3. I also added Tailwindcss with this command `npx svelte-add@latest tailwindcss` from [this ](https://github.com/svelte-add/tailwindcss)github page.
4. Then install tailwindcss with `npm install`
5. install proi-ui with this command `npm i @specialdoom/proi-ui@2.2.2`
6. Go to __layout.svelte and import necessary files as is shown in the picture above.
7. VS Code will flag an error.

**Comment by specialdoom** [comment, marked as answer]
Unfortunately there is no typescript support for proi-ui right now. Maybe in the future. You can declare inside a `global.ts` or `d.ts` as mentioned in the warning message.
